### PR TITLE
add_examples_http_poll_and_summary

### PR DESCRIPTION
**Pull request by doodyparizada**
still missing a swagger2 version.
I think that from now on I might work on the swagger2 version, and autogenerate openapi3 and change all places of `allOf` to `oneOf`

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -160,57 +175,26 @@ components:
...
-      type: object
-      properties:
-        title:
-          type: string
-        answers:
-          type: array
-          items:
-            type: string
-    MultiChoicePoll:
+    HTTPPoll:
```
Comment (Elaz) [inline openapi.yaml]: Can we simply call it “Challenge”? The fact that it’s http isn’t relevant and “Poll” is too specific.

Comment (OrenZak) [reply openapi.yaml]: @Elaz  @doodyparizada  maybe `HTMLPoll` ? I think the "Poll" postfix it's ok.

File: openapi.yaml [hunk 2]
```diff
@@ -160,57 +175,26 @@ components:
...
-            - MultiChoicePollAnswer
-        answers:
-          type: array
-          items:
-            type: integer
-    SliderPoll:
-      type: object
-      properties:
-        content_type:
+            - HTTPPoll
```
Comment (OrenZak) [inline openapi.yaml]: `HTMLPoll`

File: openapi.yaml [hunk 3]
```diff
@@ -160,57 +175,26 @@ components:
...
+        content:
           type: string
-          enum:
-            - SliderPoll
-        min:
-          type: integer
-        max:
-          type: integer
-    SliderPollAnswer:
+    HTTPPollAnswer:
```
Comment (OrenZak) [inline openapi.yaml]: `HTMLPollAnswer`

**Pull request merged**